packages/opencensus-nodejs/src/trace/instrumentation/plugin-loader.ts

```diff
@@ -140,8 +142,14 @@ export class PluginLoader {
         try {
           const plugin: Plugin = require(moduleName as string).plugin;
           this.plugins.push(plugin);
-          return plugin.enable(
-              exports, this.tracer, version, moduleConfig, basedir);
+          if (this.stats) {
```

Comment:
Changing the type signature above to make `stats` optional would also allow collapsing down the two branches of this `if`